doc/services/portability/posix/option_groups/index.rst

```diff
@@ -505,8 +505,8 @@ _XOPEN_STREAMS
 
     fattach(),
     fdetach(),
-    getmsg(),
-    getpmsg(),
+    getmsg(),  yes (will fail with ``ENOSYS``:ref:`†<posix_undefined_behaviour>`)
```

Comment:
just wondering if we need the spaces before `yes`?

```suggestion
    getmsg(),yes (will fail with ``ENOSYS``:ref:`†<posix_undefined_behaviour>`)
```

Reply:
Don't think so too.

Reply:
If needed, I will change it !